lib/qtermwidget.cpp

```diff
@@ -246,6 +246,15 @@ void QTermWidget::init(int startnow)
     m_layout->setMargin(0);
     setLayout(m_layout);
 
+#ifdef TRANSLATIONS_DIR
+    // translations
+    QString fname = QString("qtermwidget_%1.qm").arg(QLocale::system().name().left(5));
+    m_translator = new QTranslator(this);
+    qDebug() << "TRANSLATIONS_DIR: Loading translation file" << fname << "from dir" << TRANSLATIONS_DIR;
+    qDebug() << "load success:" << m_translator->load(fname, TRANSLATIONS_DIR, "_");
```

Comment:
1. isn't [this overload of load()](http://doc.qt.io/qt-5/qtranslator.html#load-1) more robust?
2. shouldn't we also consider the XDG_DATA_DIRS env. variable?